### PR TITLE
Refine the readme for training log and model saving information

### PR DESCRIPTION
**Pull request by Hsy-Intel**

### CODE REVIEW

File: cczoo/horizontal_fl/README.md [hunk 1]
```diff
@@ -77,19 +77,22 @@ Steps **②**-**⑥** will be repeated continuously during the training process.
...
 ```
 ```shell
-./start_container.sh <attestation ip addr> worker1
-/start_aesm_service.sh
+./start_container.sh worker1 <PCCS ip addr>
+./start_aesm_service.sh
 ```
 
+If running in the cloud, please modify the `PCCS server address` in the `sgx_default_qcnl.conf` file and fill in the PCCS address of the cloud.
+
```
Comment (RodgerZhu) [inline cczoo/horizontal_fl/README.md:95]: Under public cloud, is <PCCS ip addr> sitll needed? Or can it be ignored as below?
./start_container.sh worker1

File: cczoo/horizontal_fl/hfl-tensorflow/python.manifest.template [hunk 2]
```diff
@@ -72,7 +72,7 @@ fs.mount.keras.uri = "file:/root/.keras/keras.json"
 sgx.debug = false
 sgx.remote_attestation = true
```
Comment (RodgerZhu) [inline cczoo/horizontal_fl/hfl-tensorflow/python.manifest.template:73]: sgx.remote_attestation = true
can be removed.

File: cczoo/horizontal_fl/hfl-tensorflow/python.manifest.template [hunk 3]
```diff
@@ -72,7 +72,7 @@ fs.mount.keras.uri = "file:/root/.keras/keras.json"
...
 sgx.nonpie_binary = true
-sgx.enclave_size = "16G"
+sgx.enclave_size = "8G"
 sgx.thread_num = 512
 sgx.protected_files_key = "ffeeddccbbaa99887766554433221100"
```
Comment (RodgerZhu) [inline cczoo/horizontal_fl/hfl-tensorflow/python.manifest.template:77]: sgx.protected_files_key = "ffeeddccbbaa99887766554433221100"
can be removed.

**Review by yuanwu2017 — CHANGES_REQUESTED** [review]
The title is too simple. It should be "Refine the readme for training log and model saving information"

**Review by RodgerZhu — APPROVED** [review]

**Review by yuanwu2017 — APPROVED** [review]

**Pull request merged**